MediaBrowser.Model/Drawing/DrawingUtils.cs

```diff
@@ -57,6 +57,54 @@ public static class DrawingUtils
             return new ImageDimensions(newWidth, newHeight);
         }
 
+        /// <summary>
+        /// Resizes to fill box.
+        /// Returns original size if both width and height are null or zero.
+        /// </summary>
+        /// <param name="size">The original size object.</param>
+        /// <param name="fillWidth">A new fixed width, if desired.</param>
+        /// <param name="fillHeight">A new fixed height, if desired.</param>
+        /// <returns>A new size object or size.</returns>
+        public static ImageDimensions ResizeFill(
+            ImageDimensions size,
+            int? fillWidth,
+            int? fillHeight)
+        {
+            // Return original size if input is invalid.
+            if (
+                (fillWidth == null && fillHeight == null)
+                || (fillWidth == 0 || fillHeight == 0))
+            {
+                return size;
+            }
+
+            if (fillWidth == null || fillWidth == 0)
+            {
+                fillWidth = 1;
+            }
+
+            if (fillHeight == null || fillHeight == 0)
+            {
+                fillHeight = 1;
+            }
+
+            double widthRatio = (double)size.Width / (double)fillWidth;
+            double heightRatio = (double)size.Height / (double)fillHeight!;
+            // min()
```

Comment:
We do have `Math.Min` that has the exact same implementation

Reply:
I didn't find the proper documentation for Math.Min last night, and what I found implied it was int only, thanks for making me take another look :P